Updating the Firmware of your Skywallet.md

```diff
@@ -0,0 +1,61 @@
+# Updating the Firmware of your Skywallet
+
+The firmware of your Skywallet is its user interface, the software code that interacts with the hardware. And like any other software, the firmware of your Skywallet also needs periodic updations.
+
+It is important you do these updates otherwise it would put your coins under risk of security threats, as usually these firmware updates are carried out to improve the security of your Skywallet.
```

Comment:
Maybe reformulate to otherwise... you might not benefit from the newest security features and fixes. Also updates can also be made for increasing functionality.